src/App.js

```diff
@@ -30,26 +30,32 @@ class App extends Component {
     };
 
     // Uncomment this to spawn a single test item
-    //const testItem = this.spawnItem(Date.now());
-    //this.state.items.push(testItem);
+    const testItem = this.spawnItem(Date.now());
+    this.state.items.push(testItem);
 
     // Uncomment this to automatically spawn new items
     this.enableSpawner();
 
     console.log(this.state);
   }
 
-  onItemClicked = () => {
-    // Fill this in!
+  onItemClicked = (type) => {
+
+    if (type === "litter") {
+      this.setState((prevState) => ({
+        points: prevState.points + 1
+      }));
```

Comment:
Good work using a reducer function here instead of reading the previous state directly.